apps/sms/index.html

```diff
@@ -149,6 +156,33 @@ <h1 data-l10n-id="messages"></h1>
           <a id="threads-settings-button" data-icon="settings" data-l10n-id="settingsButton"></a>
         </gaia-header>
         <article id="threads-container" class="view-body edit-container" data-type="list">
+          <gaia-fast-list id="threads-list" top="0" bottom="0">
+            <!--<template header>
+              <gaia-sub-header data-time="${daySection}">${section}</gaia-sub-header>
+            </template>-->
+            <template item>
+              <li class="${classList}" data-last-message-type="${type}" data-time="${timestamp}" data-thread-id="${id}" data-draftId="${draftId}" id="thread-${id}">
+                <label class="pack-checkbox negative">
+                  <input type="checkbox" value="${id}" data-mode="${mode}">
+                  <span></span>
+                </label>
```

Comment:
I think we can remove this checkbox stuff as we will be building 'edit mode' into gaia-fast-list.